api/requirements.txt

```diff
@@ -1,4 +1,4 @@
-fastapi>=0.65.2
+fastapi>=0.73.0
```

Comment:
We should at a short reference why we have changed it behind :)
fastapi>=0.73.0  # cf. https://github.com/mindee/doctr/issues/847